test/plugins/test_sync_protocol.py

```
@@ -561,3 +569,210 @@ def test_empty_file_raises_json_error(self, tmp_path):
         f.write_text("")
         with pytest.raises(json.JSONDecodeError):
             _parse_sync_payload(str(f))
+
+
+# ===========================================================================
+# SYNC_BEHAVIOR — three hub device-write modes (Mode 3 – RECEIVE)
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)